config.py

```diff
@@ -18,7 +16,7 @@
 title_register_summary_price_text = "&pound{} inc VAT"
 
 CONFIG_DICT = {  # type: ignore
-    'DEBUG': DEBUG,
+    'LOGGING_LEVEL': os.environ['LOGGING_LEVEL'],
```

Comment:
Is there a reason this is different to the other apps where they use os.environ.get('LOGGING_LEVEL', "WARN") which would set a default if required?